CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.2.0 (2020-04-03)
+
+- Added the `args` property to the `beforeClose` event to let consumers access the provided `SkyModalCloseArgs` data. [#84](https://github.com/blackbaud/skyux-modals/pull/84)
```

Comment:
Should `event` be `event handler`?

Reply:
I don't have strong feelings here. I based that on the headings from our docs pages. We typically call them "events" and not "event handlers".

Reply:
Works for me. I don't actually know the difference. I just saw that we had "event handlers" in the title of the linked PR so I wanted to make sure. Thanks!